### ISSUE TITLE
Add basic form-based disease-phenotype entry

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/monarch-phenote#2]
First pass should have 3 fields:
1. Disease
2. Phenotype
3. Publication

(To make things simpler, as a very first pass, omit 3.)
1. The value of 1 should be anything from the disease ontology (D)
2. The value of 2 should be anything from the phenotype ontology (Ph)
3. The value of 3 will at first pass be a string (first pass) (Pub)

This should map to a graph in the following way:
- generate IRI for disease instance, `di`
- generate IRI for phenotype instance, `pi`
- create 3 triples
  - di rdf:type D
  - pi rdf:type Ph
  - di RO_0002200 pi

Check with @hdietze for modeling of Pub

**Comment by DoctorBud** [comment]
I think this ticket became closable when the original WebPhenote work was performed, and is perhaps superseded by https://github.com/monarch-initiative/monarch-phenote/issues/17, which is blocked on the review of two PRs (Noctua and bbop-client-barista).
